pkg/tanka/parse_test.go

```diff
@@ -0,0 +1,13 @@
+package tanka
+
+import (
+	"github.com/grafana/tanka/pkg/spec/v1alpha1"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestEvalJsonnet(t *testing.T) {
```

Comment:
Would love to see this also check objects, and evaluation results.

You could for example create multiple pseudo "environments" in testdata, like so:

```
- /
- /jsonnetfile.json
- /cases/
- /cases/array/main.jsonnet
- /cases/object/main.jsonnet
```

Then evaluate these as a table test, checking the error is nil and the result is what we expect.
Good work!

Reply:
Great! I'll add it. Thank you for your feedback.